compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/bir/codegen/JvmMethodGen.java

```diff
@@ -195,7 +190,7 @@
 import static org.wso2.ballerinalang.compiler.bir.codegen.JvmTerminatorGen.cleanupObjectTypeName;
 import static org.wso2.ballerinalang.compiler.bir.codegen.JvmTerminatorGen.isExternStaticFunctionCall;
 import static org.wso2.ballerinalang.compiler.bir.codegen.JvmTerminatorGen.loadChannelDetails;
-import static org.wso2.ballerinalang.compiler.bir.codegen.JvmTypeGen.loadExternalOrLocalType;
+import static org.wso2.ballerinalang.compiler.bir.codegen.JvmTypeGen.loadLocalType;
 import static org.wso2.ballerinalang.compiler.bir.codegen.JvmTypeGen.loadType;
 import static org.wso2.ballerinalang.compiler.bir.codegen.JvmTypeGen.typeOwnerClass;
 import static org.wso2.ballerinalang.compiler.bir.codegen.interop.ExternalMethodGen.genJMethodForBExternalFunc;
```

Comment:
We seemed to have a lot of static imports. This might be necessary in this place but I would like to point out this: https://stackoverflow.com/a/162195/3599535

Reply:
It is not recommended by [java](https://docs.oracle.com/javase/1.5.0/docs/guide/language/static-import.html)

Reply:
Ack. We will be cleaning this up once BString is migrated. This is one-to-one Ballerina to Java migration so thats why you are seeing static method imports.